nnfabrik/main.py

```diff
@@ -351,3 +351,77 @@ class Seed(dj.Manual):
     definition = """
     seed:   int     # Random seed that is passed to the model- and dataset-builder
     """
+
+
+def custom_nnfabrik(
+    schema,
+    use_common_fabrikant=True,
+    use_common_seed=False,
+    module_name=None,
+    context=None,
+    spawn_existing_tables=False,
+):
+    """
+    Create a custom nnfabrik module under specified DataJoint schema,
+    instantitaing Model, Dataset, and Trainer tables. If `use_common_fabrikant`
+    is set to True, the new tables will depend on the common Fabrikant table. 
+    Otherwise, a separate copy of Fabrikant table will also be prepared.
+
+    Args:
+        schema (str or dj.Schema): Name of schema or dj.Schema object
+        use_common_fabrikant (bool, optional): If True, new tables will depend on the
+           common Fabrikant table. If False, new copy of Fabrikant will be created and used. 
+           Defaults to True.
+        use_common_seed (bool, optional): If True, new tables will depend on the
+           common Seed table. If False, new copy of Seed will be created and used. 
+           Defaults to False.
+        module_name (str, optional): Name property of the returned Python module object.
+            Defaults to None, in which case the name of the schema will be used.
+        context (dict, optional): If non None value is provided, then a module is not created and
+            instead the tables are defined inside the context.
+        spawn_existing_tables (bool, optional): If True, perform `spawn_missing_tables` operation
+            onto the newly created table. Defaults to False.
+
+    Returns:
+        Python Module object or None: If `context` was None, a new Python module containing 
+            nnfabrik tables defined under the schema. The module's schema property points 
+            to the schema object as well. Otherwise, nothing is returned.
+    """
+    if isinstance(schema, str):
```

Comment:
Maybe also raise an error if schema is not a string but a `dj.schema` instead of a `CustomSchema`

Reply:
actually I want to leave that a choice for the user. They can elect to use this against a regular schema.